src/Sylius/Bundle/AdminBundle/Resources/config/form/events.yml

```
@@ -0,0 +1,32 @@
+sylius_ui:
+    events:
+        sylius.admin.admin_user.form_content:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/AdminUser/_form_content.html.twig'
```

Use camelCase with `_` prefix in the file names
```suggestion
                    template: '@SyliusAdmin/AdminUser/_formContent.html.twig'
```
or
```suggestion
                    template: '@SyliusAdmin/AdminUser/Form/_content.html.twig'
```

---

@GSadee Symfony recommends snake case for file and directory names: https://symfony.com/doc/current/templates.html#template-naming

---

`template: '@SyliusAdmin/AdminUser/Form/_content.html.twig'` it is therefore :)

---

@vvasiloi yup, I'm aware of it, but in Sylius we are using camelCase for template names for ages, and I would rather be consistent with what we have in Sylius, but this is still our unwritten convention

---

@GSadee  Yes, I know that, I was just reasoning the authors decision.
Maybe it's time to make it a written convention, enforced with some linting.